src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
     }
```

Comment:
Avoid using static methods as it is considered a bad practice in object-oriented programming. Instead, you can create an instance of `ColorSupplier` in the class where you need to use it.